spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/connecting.adoc

```diff
@@ -15,7 +15,7 @@ To close existing Consumers, call `stop()` (and then `start()`) on the `KafkaLis
 For convenience, the framework also provides an `ABSwitchCluster` which supports two sets of bootstrap servers; one of which is active at any time.
 Configure the `ABSwitchCluster` and add it to the producer and consumer factories, and the `KafkaAdmin`, by calling `setBootstrapServersSupplier()`.
 When you want to switch, call `primary()` or `secondary()` and call `reset()` on the producer factory to establish new connection(s); for consumers, `stop()` and `start()` all listener containers.
-When using `@KafkaListener` s, `stop()` and `start()` the `KafkaListenerEndpointRegistry` bean.
+When using `@KafkaListener`+++s+++, `stop()` and `start()` the `KafkaListenerEndpointRegistry` bean.
```

Comment:
in AsciiDoc, `\`s` will be rendered as
<img width="421" alt="Screenshot 2023-10-29 at 21 17 19" src="https://github.com/spring-projects/spring-kafka/assets/8211458/86b563f9-a9dc-4988-9336-f27480783cab">

that was why a space was inserted to avoid it. however, it looks weird:
<img width="312" alt="Screenshot 2023-10-29 at 21 18 08" src="https://github.com/spring-projects/spring-kafka/assets/8211458/8ec5d961-ae84-4e4c-b996-bddc3af19fa1">

using `+++` solves the issue:
<img width="266" alt="Screenshot 2023-10-29 at 21 20 01" src="https://github.com/spring-projects/spring-kafka/assets/8211458/6f05f0e9-cbc1-4c4c-83aa-698be5717dcd">